app/Models/Permission.php

```diff
@@ -96,6 +96,8 @@ class Permission extends Model implements Validatable
 
     public const ACTION_SETTINGS_RENAME = 'settings.rename';
 
+    public const ACTION_SETTINGS_DESCRIPTION = 'settings.description';
```

Comment:
New permissions need to be added to `permissions` array:

https://github.com/pelican-dev/panel/blob/82b0aff105730ce5b8bd410756c72c97c582e14a/app/Models/Permission.php#L221-L227